cmake/multi_image.cmake

```diff
@@ -352,7 +352,7 @@ function(add_child_image_from_source)
         file(
           APPEND
           ${preload_file}
-          "set(${shared_var} ${${shared_var}} CACHE INTERNAL \"NCS child image controlled\")\n"
+          "set(${shared_var} \"${${shared_var}}\" CACHE INTERNAL \"NCS child image controlled\")\n"
```

Comment:
I was not aware of that warning. In either case it would be nice if it could be supported. It seems to work in other cases

Reply:
> I was not aware of that warning. In either case it would be nice if it could be supported. It seems to work in other cases

I assume it originates from old times, especially when msys /minGW was still used.
Generally Zephyr supports space separated arguments, like `-DCONF_FILE="file1.conf file2.conf"`, then spaces in paths are discouraged, but it seems it could be valuable to see if toolchains (and tools in general) should be considered safe to install in paths containing spaces.